api/src/main/resources/workbench-api.yaml

```diff
@@ -1584,6 +1584,21 @@ paths:
           description: Internal Error
           schema:
             "$ref": "#/definitions/ErrorResponse"
+  "/v1/monitoring/billingProjectBufferStatusByTier":
```

Comment:
I'm not sure the other method was actually being used anywhere, so I don't know if you need this one.

The thing that feeds into the alerts / dashboards is coming from `updateGaugeMetrics`. That part may need updating. From digging through the archives, it looks like the original purpose of this endpoint was to poke at the buffer status during perf testing: https://github.com/all-of-us/workbench/pull/2845 , however I don't see any checked in references to this endpoint (here or in mcnulty), so it may be subsumed by the updated monitoring dashboards and can likely be removed.

Gauge metrics integration happens here: https://github.com/all-of-us/workbench/blob/01a62968fa8d566823cc0e21e97021eb67174ec6/api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java#L91-L96

Reply:
Good catch, thanks.  There are a variety of Gauge Metrics which could be affected here, so I filed https://precisionmedicineinitiative.atlassian.net/browse/RW-6137

Reply:
I could imagine a debugging use for these monitoring endpoints (even if we don't actually use them now) so I'm inclined to keep them for now